crates/ruff_python_formatter/resources/test/fixtures/black/py_312/type_aliases.py

```diff
@@ -0,0 +1,5 @@
+type A=int
```

Comment:
Why no long cases here?

We're also missing quite a few cases. Perhaps https://github.com/RustPython/Parser/blob/704eb40108239a8faf9bd1d4217e8dad0ac7edb3/parser/src/parser.rs#L901-L936 would be helpful